### DISCUSSION TITLE
How to keep PopupWindow shown when clicked on it

### DISCUSSION THREAD

**Discussion by iincity** [slint-ui/slint#5301] [General]
How to keep PopupWindow shown when clicked on it and closed when clicked close button on it

**Comment by ghost** [comment]
Hi,

I think this code should work for you 

https://slintpad.com/?snippet=import+%7B+Button+%7D+from+%22std-widgets.slint%22%3B%0A%0Aexport+component+Example+inherits+Window+%7B%0A++++width%3A+400px%3B%0A++++height%3A+400px%3B%0A%0A++++popup+%3A%3D+PopupWindow+%7B%0A++++++++close-on-click%3A+false%3B%0A++++++++Rectangle+%7B+%0A++++++++++++height%3A100%25%3B+width%3A+100%25%3B+background%3A+blue%3B+%0A++++++++++++Button+%7B%0A++++++++++++++++text%3A+%22close%22%3B%0A++++++++++++++++clicked+%3D%3E+%7B%0A++++++++++++++++++++popup.close%28%29%3B%0A++++++++++++++++%7D%0A++++++++++++%7D%0A++++++++%7D%0A++++++++x%3A+20px%3B+y%3A+20px%3B+height%3A+100px%3B+width%3A+200px%3B%0A++++%7D%0A%0A++++TouchArea+%7B%0A++++++++height%3A100%25%3B+width%3A+100%25%3B%0A++++++++clicked+%3D%3E+%7B+popup.show%28%29%3B+%7D%0A++++%7D%0A%7D&style=native
